reco_utils/aml/svd_training.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import sys
+
+sys.path.append("../../")
+
+import argparse
+import os
+import pandas as pd
+import surprise
+
+try:
+    from azureml.core import Run
+    HAS_AML = True
+    run = Run.get_context()
+except ModuleNotFoundError:
+    HAS_AML = False
+
+from reco_utils.evaluation.python_evaluation import *
+from reco_utils.recommender.surprise.surprise_utils import compute_predictions, compute_all_predictions
+
+def svd_training(args):
```

Comment:
I think we should take a look at this together. I'm doing some refactoring in SVD due to the benchmarks and I'm creating some functions. This is an example https://github.com/Microsoft/Recommenders/blob/benchmark_feb/reco_utils/recommender/surprise/surprise_utils.py. In `svd_training` we might want to split the data loading, training and evaluation. I'm doing that for the benchmarks https://github.com/Microsoft/Recommenders/blob/benchmark_feb/notebooks/03_evaluate/comparison_movielens.ipynb

Reply:
Feel free to refactor this file and merge your utils to this branch.

Reply:
I suggest we address this in a separate issue / PR.

Reply:
agree